src/util/comparison.ts

```diff
@@ -56,15 +85,30 @@ export function selectMostSimilarChapter(original: Chapter, options: Chapter[]):
 export function getNumberUnreadChapters(chapterList: Chapter[]): number {
   let highestRead = 0;
   let highestReleased = 0;
+  let previousChapNumber = 0;
+  let cumulativeGaps = 1;
 
-  chapterList.forEach((chapter: Chapter) => {
+  const chapters = consolidateAndSortChapters(chapterList);
+
+  chapters.forEach((chapter: Chapter, index: number) => {
+    let absoluteNumber = cumulativeGaps + index;
     const chapterNumber = parseFloat(chapter.chapterNumber);
-    if (chapter.read && chapterNumber > highestRead) {
-      highestRead = chapterNumber;
+
+    const gap = Math.ceil(chapterNumber - previousChapNumber) - 1;
+    if (gap > 1) {
+      // A gap between chapters was found. Account for this in the absolute numbers
+      absoluteNumber += gap;
```

Comment:
nit: you could avoid needing to increment both here and only set `absoluteNumber` after this block (since it's not used yet)

Reply:
Ahh yes, didn't catch that